backend/regalloc/regalloc_validate.ml

```diff
@@ -495,7 +496,8 @@ end = struct
            interference graph, which can lead to cycles. *)
         ()
       | _ ->
-        if instr.desc <> old_instr.desc
+        (* CR-soon xclerc for xclerc: avoid polymorphic equality. *)
```

Comment:
why did you leave this for later?

Reply:
Laziness, mainly, the comparison function
will be annoying to write.